rules/web/web_certificate_authority_common_whitelist.yml

```
@@ -0,0 +1,62 @@
+title: CA issuer whitelist
+description: Detects all other CA issuers other than listed
+author: Yulli Chong - InfoSec Innovations
+date: 2019/01/11
+status: experimental
+references:
+    - https://www.godaddy.com/help/verify-domain-ownership-html-or-dns-7452
+    - https://www.alexa.com/topsites
+    - https://www.pluralsight.com/blog/software-development/top-reliable-ssl-certificates
+
+logsource:
+    category: ssl
+detection:
+    selection:
+      CA Issuer:
```

As this is a new log source, I suggest to minimize this name to *Issuer*. This should be clear enough.
```suggestion
      Issuer:
```